Speedykom/Regional-Pandemic-Analytics
# Feat: Add email sending feat to new created users

An admin can create a new user from the frontend by clicking on New User Button.
However the create password is randomly generated and the user has no idea about it. 
To fix this issue, when a user is created, an email is sent to him containing the username and the password. The first time he logs in, he is obliged to create his own specific password.
![image](https://github.com/Speedykom/Regional-Pandemic-Analytics/assets/165824500/347c4cd4-7597-480d-a248-908a601b3330)

![image](https://github.com/Speedykom/Regional-Pandemic-Analytics/assets/165824500/a3bb6b6c-ce34-4ac3-99c2-7d3ffef8204a)

- The URL will be changed to the dev domain or the prod domain
- The same email will be sent in French if the admin who created the new user is using French in RePan 

![image](https://github.com/Speedykom/Regional-Pandemic-Analytics/assets/165824500/1449e2a4-663d-4ccd-bb63-9cc532c129a8)


**Test Scenario**
1. Create a user with a valid email address
2. Check the email for the credentials 
3. Logs in and change the password
